binary/type.proto

```diff
@@ -30,242 +30,104 @@ message Type {
     }
 
     message I8 {
-        PhysicalType physical_type = 1;
-
-        enum PhysicalType {
-            SYSTEM_DEFAULT = 0;
-        }
-    }
-
-    message U8 {
-        PhysicalType physical_type = 1;
-
-        enum PhysicalType {
-            SYSTEM_DEFAULT = 0;
-        }
+        Variation variation = 1;
```

Comment:
I'm no protobuf expert but can `variation` be made optional?  What is the equivalent of `SYSTEM_DEFAULT` from before?

Reply:
In proto 3, all items are optional. So system default would be an undefined variation (at least that was my thinking).